### PR TITLE
Add links to accounting KPIs

### PR DESCRIPTION
**Pull request by choldgraf**
This adds two links to our accounting KPIs page, now that we track this publicly.

- closes https://github.com/2i2c-org/team-compass/issues/560

### CODE REVIEW

**Review by damianavila — APPROVED** [review]
LGTM, although wondering why this is http instead of https?

**Comment by choldgraf** [comment]
Good call - i just realized we weren't enforcing https in GitHub pages so I have now turned that on. We can switch these to https but I can't because I'm a phone now

File: organization/kpis.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Key Performance Indicators
 
 We use **Key Performance Indicators** to measure and track our progress and impact as an organization.[^1]
+These are tracked along with other important metrics in [our KPIs dashboard site](http://2i2c.org/kpis/).
+
+```{button-link} http://2i2c.org/kpis/
```
Comment (damianavila) [inline organization/kpis.md:6]:
```suggestion
```{button-link} https://2i2c.org/kpis/
```

File: organization/kpis.md [hunk 2]
```diff
@@ -71,10 +77,10 @@ They represent a positive outcome that is aligned with our mission as a result o
   - Source
 - - Monthly revenue
   - Monthly revenue, broken down by category (grants, development contracts, cloud service contracts)
-  - See this issue to track this effort: https://github.com/2i2c-org/team-compass/issues/560
+  - See [our Accounting KPIs page](http://2i2c.org/kpis/finances.html)
```
Comment (damianavila) [inline organization/kpis.md:80]:
```suggestion
  - See [our Accounting KPIs page](https://2i2c.org/kpis/finances.html)
```

File: organization/kpis.md [hunk 3]
```diff
@@ -71,10 +77,10 @@ They represent a positive outcome that is aligned with our mission as a result o
...
 - - Monthly operational costs
   - Monthly operational costs, broken down by category (engineering, partnerships and community, executive, fiscal fee, services)
-  - See this issue to track this effort: https://github.com/2i2c-org/team-compass/issues/560
+  - See [our Accounting KPIs page](http://2i2c.org/kpis/finances.html)
```
Comment (damianavila) [inline organization/kpis.md:83]:
```suggestion
  - See [our Accounting KPIs page](https://2i2c.org/kpis/finances.html)
```

**Pull request merged**